### PR TITLE
Refactor Comments form

### PR DESCRIPTION
**Pull request by tttzach**
Refactor comments form to include a Display button to get comments instead of hitting enter which can be unreliable at times.
Screenshot 2020-06-15 at 1 41 32 PM

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -41,15 +41,18 @@ <h1>definitely not zachary's website.</h1>
...
+      <form method="POST" onsubmit="return loadComments(max.value, language.value)">
+      <p>Load how many comments: <input type="int" id="max" name="max" required>
+      <select id="language" name="language" required>
+        <option value="es" selected>English</option>
         <option value="zh">Chinese</option>
         <option value="es">Spanish</option>
         <option value="hi">Hindi</option>
         <option value="ar">Arabic</option>
       </select>
+      <input type="submit" name="display" value="Display" />
```
Comment (gbuenoandrade) [inline portfolio/src/main/webapp/index.html:53]: Nit: indentation

File: portfolio/src/main/webapp/comments.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ function loadComments(maxInt, language) {
         commentListElement.appendChild(createCommentElement(comment));
     })
   });
+  return false;
```
Comment (gbuenoandrade) [inline portfolio/src/main/webapp/comments.js:36]: What does `false` mean in this context?

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (tttzach) [reply portfolio/src/main/webapp/index.html]: Fixed, thanks!

File: portfolio/src/main/webapp/comments.js (hunk 2 above)
Comment (tttzach) [reply portfolio/src/main/webapp/comments.js:37]: `false` here is to ensure that the `onsubmit` function called by the form does not [redirect](https://stackoverflow.com/questions/35037069/what-is-the-meaning-of-onsubmit-return-false-javascript-jquery). I have added a comment to clarify this. Do let me know if there's a better way to do this!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (tttzach) [reply portfolio/src/main/webapp/index.html]: `RESOLVED`

File: portfolio/src/main/webapp/comments.js (hunk 2 above)
Comment (gbuenoandrade) [reply portfolio/src/main/webapp/comments.js:37]: Maybe there is, but I'm not aware, so looks good to me hehe

Comment (tttzach) [reply portfolio/src/main/webapp/comments.js:37]: Ok, I'll keep an eye out for better ways to do this for the capstone

Comment (tttzach) [reply portfolio/src/main/webapp/comments.js:37]: `RESOLVED`

**Pull request merged**